src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,21 +2,25 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.debug("Method login was called. Params: login={}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
+            logger.error("Authentication failed for user: {}", login);
```

Comment:
The error log should not disclose user login details as it may lead to security issues. Consider logging an internal user ID or a masked version of the login if necessary for troubleshooting.